packages/jaeger-ui/src/components/SearchTracePage/SearchForm.js

```diff
@@ -369,6 +374,16 @@ export class SearchFormImpl extends React.PureComponent {
           </Field>
         </FormItem>
 
+        {selectedLookback !== 'custom' && (
```

Comment:
I have a suggestion for this part of the UI. 
My demo here:
https://codesandbox.io/s/basic-antd-4-22-6-forked-301xpo?file=/demo.js:270-899

Reply:
Used a popover instead similar to what's used for the `Tag` field, also avoids importing a new package.